Test/git-issues/git-issue-645.dfy

```diff
@@ -1,4 +1,4 @@
 // dafny should emit exit value 1
-// RUN: %dafny_0 /xyz    > "%t"
+// RUN: %dafny_0 /xyz    > "%t" || true
```

Comment:
I though `dafny_0` was introduced to avoid having to add `||true`.  I like `||true`, but then do we still need `%dafny_0`?

Reply:
dafny_0 adds the argument `/countVerificationErrors:0`. In this PR that argument no longer swallows errors in command line arguments, but only in verification errors.

Reply:
I thought this option was deprecated already. If not, it certainly needs renaming. I think the only use was to not have expected verification errors in tests cause the test to fail.